lib/create_send_rails/deliverer.rb

```diff
@@ -2,29 +2,60 @@ module CreateSendRails
   class Deliverer
     attr_accessor :settings
 
+    SAILTHRU = 'sailthru'.freeze
+    CREATSEND = 'creatsend'.freeze
+
     def initialize(values)
       self.settings = {}.merge(values)
     end
 
     def deliver!(mail)
       @mail = mail
 
+      delivery_system == SAILTHRU ? send_via_sailthru : send_via_createsend
+    end
+
+    private
+
+    attr_reader :mail
+
+    def sailthru_auth
```

Comment:
Do we need sailthru_auth?

Reply:
no, this should be just `auth`- it's a glitch from our session